tests/Test/Prometheus/AbstractCounterTest.php

```diff
@@ -58,6 +58,80 @@ public function itShouldIncreaseWithLabels()
         );
     }
 
+    /**
+     * @test
+     */
+    public function itShouldIncreaseWithLabelsAndDefaults()
```

Comment:
Maybe add the `test` prefix to all test methods. Also add the `void` return type to all of them. We could also add the @covers annotation for code coverage generation.

Reply:
Test prefix is redundant since `@test` exists. Adding any return type makes it incompatible with older PHP versions.